app/assets/stylesheets/content/_forms.sass

```diff
@@ -57,6 +57,8 @@ $form--field-types: (text-field, text-area, select, check-box, radio-button, ran
     padding: 30px 20px
     background-color: $content-form-bg-color
     border: $content-form-border
+  &.-higher
+    min-height: 31em
```

Comment:
i cannot judge if this is acceptable (because i'm not familiar with the rest of the frontend code). but it's probably a source of confusion, headaches, and bugs once anyone changes something. (not that the rest of openproject is any better, though)